configure.py

```diff
@@ -414,7 +414,7 @@ def DolphinLib(lib_name, objects):
             Object(NonMatching, "d/d_snap.cpp"),
             Object(Matching,    "d/d_point_wind.cpp"),
             Object(NonMatching, "d/actor/d_a_agb.cpp"),
-            Object(NonMatching, "d/actor/d_a_arrow.cpp"),
+            Object(Matching,    "d/actor/d_a_arrow.cpp", cflags=[*cflags_framework, "-sym off"]),
```

Comment:
This should just be `-sym off`, right? cflags_rel already includes cflags_framework

Reply:
arrow isn't it a rel, it's an actor in main.dol.
If I do `cflags=["-sym off"]` it won't compile at all, it seems to replace all the flags with just "-sym off", instead of adding them.

Reply:
ah. we can fix that I suppose, but for now